DuggaSys/courseed.php

```diff
@@ -140,12 +148,15 @@
 				</div>
     		</div>
     		<div style='float:right; padding-top:20px; width: 464px;' >
-    			<input id='saveCourse' class='submit-button' type='button' value='Save' title='Save changes' onclick="validateForm('editCourse')" />
+    			<input id='saveCourse' class='submit-button' type='button' value='Save' title='Save changes' onclick="showLoader(); setTimeout(function() {validateForm('editCourse');hideLoader(); }, 1000); "/>
```

Comment:
Would it be possible to have the loader disappear after the box has closed instead of a 1 second timer?

Reply:
It does. The timer just delays both 'validateForm('editCourse')' and 'hideLoader()'. 'validateForm('editCourse')' closes the window when it is done, and 'hideLoader()' runs directly after. Putting it before can make it disappear before it is done validating. However, I can change the time to 0.1 seconds instead, and it will work.